### PR TITLE
Team3 individual skills section

### PR DESCRIPTION
**Pull request by liux2**
This Pull Request adds the content needed for the individual section of the Assessment Guide.

### CODE REVIEW

**Comment by lussierc** [comment]
This looks good. Thanks for adding the changes suggested by @Jordan-A in Issue #15!

File: README.md
Comment (enpuyou) [inline README.md:70]: Can you be more specific about what the "additional commands and flags" actually are? Other students might be confused about what "additional commands" they should run in order to achieve the level of  :sunglasses:

**Review by enpuyou — CHANGES_REQUESTED** [review]
There is an inconsistency in the use of "Excellent", "E", and what your team has previously used in other sections of assessment <https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-assessment/pull/7>

> > The content looks great! One more point that might worth discussing:
> > Your team added an equal sign after each emoji and a further explanation on the highest level requirement of each assessment while a couple of other teams have not. Should there be a more specific standard for the assessment then?
> 
> @enpuyou We followed the Markdown standard that all teams should be following with regards to the emoji's and `=` signs. This standard can be found in [this file](https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown/blob/master/AssessmentFormat.md) which is apart of the Markdown repository for the course. All teams should be following this standard as far as I know. With regards to the higher level explanation, this appeared in the README examples in the Assessment repository and the previously mentioned Markdown standard file also mentions that a detailed paragraph describing the subcategory should be included.

File: README.md
Comment (lussierc) [reply README.md:70]: @enpuyou Hi Enpu. This content was already included originally in the repository by the instructor. The only thing we changed was the letter criteria system to the emoji's, whereas the text is still from the course instructor.

As a result, I don't think it is necessary to change this text.

**Comment by lussierc** [comment]
> There is an inconsistency in the use of "Excellent", "E", and what your team has previously used in other sections of assessment #7
> 
> > > The content looks great! One more point that might worth discussing:
> > > Your team added an equal sign after each emoji and a further explanation on the highest level requirement of each assessment while a couple of other teams have not. Should there be a more specific standard for the assessment then?
> > 
> > 
> > @enpuyou We followed the Markdown standard that all teams should be following with regards to the emoji's and `=` signs. This standard can be found in [this file](https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown/blob/master/AssessmentFormat.md) which is apart of the Markdown repository for the course. All teams should be following this standard as far as I know. With regards to the higher level explanation, this appeared in the README examples in the Assessment repository and the previously mentioned Markdown standard file also mentions that a detailed paragraph describing the subcategory should be included.

Hi @enpuyou, I was wondering where this was specifically used? I cannot seem to find this issue.

File: README.md
Comment (enpuyou) [inline README.md]: Here

File: README.md
Comment (enpuyou) [inline README.md]: Here

File: README.md
Comment (enpuyou) [inline README.md]: Here

File: README.md
Comment (lussierc) [reply README.md]: Ah, thank you @enpuyou.

File: README.md
Comment (lussierc) [reply README.md]: Thanks, I fixed it @enpuyou.

File: README.md
Comment (lussierc) [reply README.md]: Thanks, I fixed it @enpuyou.

File: README.md
Comment (lussierc) [reply README.md]: Thanks, I fixed it @enpuyou.

**Review by enpuyou — APPROVED** [review]

**Review by corlettim — APPROVED** [review]

**Review by enpuyou — APPROVED** [review]

**Comment by enpuyou** [comment]
@liux2 @lussierc I'd like to have two more members from your group to review this PR and approve if it's believed to be up to standard

**Review by lussierc — APPROVED** [review]
I am approving this PR. The PR accurately updates the README with the new emoji scale. Additionally, it also offers what seem to be reasonably descriptive and informative content for the Individual Skills section, without noticeable issues. I also will note that Team 3 has taken steps to make improvements based on the Technical Leaders suggestions.

**Review by ibarrezueta — APPROVED** [review]

File: assessment.md
Comment (enpuyou) [inline assessment.md]: As I am looking at this again, there is another change needs to be made before merging. The use of emoji here is incorrect as it should be in the format like this `:disappointed:` instead of "😞". It should be good to go after this issue is resolved. I'm sorry for this going back and forth!

**Review by enpuyou — CHANGES_REQUESTED** [review]

**Review by enpuyou — APPROVED** [review]

File: assessment.md
Comment (lussierc) [reply assessment.md]: @enpuyou, interesting, I see what you mean. I just changed them!

**Pull request merged**
